content/en/continuous_integration/guides/test_configurations.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Test Configurations
+kind: guide
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">CI Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+Tests evaluate the behavior of code for a set of given conditions. Some of those conditions are related to the runtime and settings of the system where the tests are run. The same code run in different conditions can have a different behavior, so developers usually configure their tests in order to run in different conditions and validate that the behavior is the expected for all of them.
```

Comment:
@nachoBonafonte I'm realizing that here we talk about "conditions" and later we talk about "configurations" without describing what's the relationship between them. What about doing something like this (pending editorial review of course):

```suggestion
Tests evaluate the behavior of code for a set of given conditions. Some of those conditions are related to the environment where the tests are run, such as the operating system or the runtime used. The same code executed different sets of conditions can behave differently, so developers usually configure their tests to run in different sets of conditions and validate that the behavior is the expected for all of them. A specific set of conditions is what we call a "configuration".
```